File: 0167-two-sum-ii-input-array-is-sorted/0167-two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, k: List[int], target: int) -> List[int]:
        l,r=0,len(k)-1
        while l<r:
            curSum=k[l]+k[r]
            if curSum>target:
                r-=1
            elif curSum<target:
                l+=1
            else:
                return [l+1,r+1]

File: 0167-two-sum-ii-input-array-is-sorted/0167-two-sum-ii-input-array-is-sorted.java
class Solution {
    public int[] twoSum(int[] k, int target) {
        int l=0;
        int r=k.length-1;
        while(l<r)
        {
          int curSum=k[l]+k[r];
            if(curSum>target)
            {
                r--;
            }
            else if(curSum<target)
            {  
                l++;
            }
            else
            {
                return new int[]{l+1,r+1};
                }
    }
                    return new int[]{};
}
}